File: ServicioBarberia/src/main/java/com/ceiba/barberia/dominio/entidades/Cita.java
package com.ceiba.barberia.dominio.entidades;

import java.util.Date;

import lombok.AllArgsConstructor;
import lombok.Builder;
import lombok.Data;
import lombok.NoArgsConstructor;

@Data
@NoArgsConstructor
@AllArgsConstructor
@Builder
public class Cita {
	
	private Long id;
	private Date fecha;
	private Barbero barbero;
	private Boolean corteCabello;
	private Boolean corteBarba;
	private Boolean lavado;
	private String nombreCliente;
}

File: FrontBarberia/src/app/core/model/cita.ts
import { Barbero } from './barbero';

export class Cita {

    constructor(
        public id:number,
        public fecha:Date,
        public barbero:Barbero,
        public corteCabello:boolean,
        public corteBarba:boolean,
        public lavado:boolean,
        public nombreCliente:string
    ) {}
}